hack/e2e/run.sh

```diff
@@ -76,6 +77,8 @@ TEST_EXTRA_FLAGS=${TEST_EXTRA_FLAGS:-}
 EBS_INSTALL_SNAPSHOT=${EBS_INSTALL_SNAPSHOT:-"false"}
 EBS_INSTALL_SNAPSHOT_VERSION=${EBS_INSTALL_SNAPSHOT_VERSION:-"v6.1.0"}
 
+HELM_TEST=${HELM_TEST:-"false"}
```

Comment:
I feel like `HELM_TEST` is ambiguous - maybe we should consider something like `HELM_CT_TEST`?